stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/utils/extensions/Channel.kt

```diff
@@ -125,8 +125,8 @@ internal fun Channel.getLastMessagePreviewText(
     }
 }
 
-private fun getAttachmentPrefix(context: Context, attachment: Attachment): String? =
+private fun getAttachmentPrefix(attachment: Attachment): String? =
     when (attachment.type) {
-        ModelType.attach_giphy -> context.getString(R.string.stream_last_message_attachment_giphy)
+        ModelType.attach_giphy -> "/giphy"
```

Comment:
Not a localized string anyway, moved to hardcoded string to avoid having to share the string resource with the old UI impl